### ISSUE TITLE
NWB DynamicTable --> MATLAB Table converter

### ISSUE THREAD

**Issue by bendichter** [NeurodataWithoutBorders/matnwb#111]
- [x]  take as input an NWB table object and output an MATLAB table
- [ ]  parse indexed column for columns that have varying length arrays
- [ ]  parse `DynamicTableRegion`s (and indexed `DynamicTableRegion`s)
- [ ]  tests
- [ ]  documentation
	- [ ]  function comments
	- [ ]  incorporating into tutorials

**Comment by ln-vidrio** [comment]
With the inclusion of Ragged Arrays, the height constraint for MATLAB tables is broken.  May need to switch to containers.Map with arrays, struct arrays, or a struct with arrays.

**Comment by bendichter** [comment]
@cechava are all of these items complete now?

**Comment by cechava** [comment]
yes, will close this issue when I merge PR #320 which is the tutorial covering its usage.

**Comment by lawrence-mbf** [comment]
#320 has since been merged.
